tests/drivers/disk/disk_performance/testcase.yaml

```diff
@@ -0,0 +1,11 @@
+tests:
+  drivers.disk.usdhc:
+    harness: ztest
+    harness_config:
+      fixture: fixture_sdhc
+    filter: CONFIG_SDMMC_USDHC
```

Comment:
Why is it filtered? Test does not seem to be platform specific.

Reply:
It doesn't need to be. I'll remove the filter and add `tags: sdhc` to ensure the test is only run on devices with a disk present.